src/sentry/snuba/entity_subscription.py

```diff
@@ -293,6 +327,31 @@ def get_entity_extra_params(self) -> Mapping[str, Any]:
             "granularity": self.get_granularity(),
         }
 
+    def aggregate_query_results(
+        self, data: List[Dict[str, Any]], alias: Optional[str] = None
+    ) -> List[Dict[str, Any]]:
+        value_col_name = alias if alias else "value"
+
+        aggregated_data: Dict[str, Any] = {}
+        session_status = tag_key(self.org_id, "session.status")
```

Comment:
`tag_key` could throw a `MetricIndexNotFound` exception here as well, though it is unlikely if the call did not throw in subscription processor.

Reply:
So this could be problematic, when can this happen?